static/css/main.css

```diff
@@ -31,78 +31,74 @@ figure {
   margin: 0;
 }
 
-a {
-  color: #279e6a;
-  color: #279e6aee;
+
+.btn {
+  display: block;
+  color: inherit;
   text-decoration: none;
-  transition: color .3s;
+  background-color: #f9f9f9;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  padding: 3px 15px;
+  transition: all .3s ease;
 }
-a:hover {
-  color: #00bf6b;
-  color: #00bf6bee;
+.btn:hover {
+  color: #000;
+  background-color: #eee;
 }
 
 /* Header */
 body > header {
   position: fixed;
   top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
   width: 100%;
   padding: 1rem;
-  background-color: rgba(255, 255, 255, .98);
   font-family: Arial, Helvetica, sans-serif;
+  background-color: rgba(255, 255, 255, .98);
   box-shadow: 0 2px 2px rgba(50, 50, 50, .2);
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
   z-index: 9;
 }
+  
 body > header h1 {
   margin: 0;
   font-weight: 300;
   font-size: 3rem;
 }
+
 body > header h1 a {
   padding: 1rem;
+  color: inherit;
   text-decoration: inherit;
-  color: #333;
-  text-decoration: inherit;
-}
-
-body > header nav {
-  float: right;
-  margin: 0 2rem 0 0;
-  display: block;
-}
-body > header nav a {
-  padding: 3px 15px;
-  font-size: .9em;
-  text-decoration: none;
+  transition: all .3s ease;
```

Comment:
Applied [here](https://github.com/neomad-team/neomad.org/blob/master/static/css/main.css#L234). Though it might not be the best selector to apply it on as it makes a fancy load resize.